modules/templates/helper.go

```diff
@@ -278,6 +281,36 @@ func renderCommitMessage(msg string, opts markup.RenderIssueIndexPatternOptions)
 	return template.HTML(msgLines[0])
 }
 
+func RenderCommitBody(msg, urlPrefix string, metas map[string]string) template.HTML {
+	return renderCommitBody(msg, markup.RenderIssueIndexPatternOptions{
+		URLPrefix: 	urlPrefix,
+		Metas:		metas,
+	})
+}
+
+func RenderCommitBodyLink(msg, urlPrefix string, urlDefault string, metas map[string]string) template.HTML {
```

Comment:
This function does not seems to be used anywhere, so should be dropped

Reply:
Sounds good! I just copied the `RenderCommit` functions and changed them a tiny bit.